manifests/ops-files/set-fs-inotify-limit.yml

```diff
@@ -0,0 +1,19 @@
+- type: replace
+  path: /addons/-
+  value:
+    jobs:
+      - name: sysctl
```

Comment:
What else does this bring in? It seems excessive to bring in a whole new release for one configuration option.

Reply:
I believe the intention was to use pre-existing cloudfoundry tooling instead of trying to redo something in a specific way.

Reply:
@tvs 

Writing a new bosh release requires a lot of infrastructure setup(blob stores, pipelines and testing and a release process), which seem unnecessary given a existing release exists. Os-conf-release mostly has a bunch of os odds and ends and is convenient for running short amounts of config without creating new releases. 

Something to point out is that os-conf-release is only 17 KB. It's not a heavy weight release by any measure. Some people use it for simple things like creating users on their systems. 

We still need to do some more investigation about what is causing this issue, but if something like this isn't included, if you're trying to watch files (which a lot of people do with fluent bit and the tail plugin) at some point based on some factors we havn't determined, but are likely scale and workload dependent you will stop receiving logs. I suspect, but am highly unsure of, that log files are not being properly reaped or rotated, and are accumulating.

 Given a lot of customers use this type of egress for compliance, legal, and security reasons(see how syslog release forwards bosh logs), it seems important that the log egress isn't interrupted. So while this may seem like a "small configuration option" I don't believe it's a trivial one.

Reply:
My point wasn't to write a standalone release for this - that just creates the same problem. My point was that this is just running a small few `sysctl` commands - the same thing could have been baked into `kubo-release` without incurring all of that extra overhead you mention.

Reply:
I figured given the workaround nature it was better to not at the moment add it directly to the kubo release itself. Again, I'd rather there be more comprehensive solutions(and even then, those might come from things like logrotation managment releases).